chopper/lib/src/base.dart

```diff
@@ -171,23 +171,19 @@ class ChopperClient {
     if (service == null) {
       throw Exception('Service of type \'$serviceType\' not found.');
     }
-    return service;
+    return service as ServiceType;
   }
 
-  Future<Request> _encodeRequest(Request request) async {
-    if (converter != null) {
-      return converter.convertRequest(request);
-    }
-
-    return request;
+  Future<Request?> _encodeRequest(Request request) async {
```

Comment:
`Future<Request>` non nullable should be possible

Reply:
Yes. Removed '?'